src/languages/rust.js

```diff
@@ -216,7 +223,7 @@ export default function(hljs) {
         begin: [
           /fn/,
           /\s+/,
-          hljs.UNDERSCORE_IDENT_RE
+          UNDERSCORE_IDENT_RE
```

Comment:
Maybe should we apply `(r#)?` only on `className: variable`?

L226 here, className is title.function so user normally not use `r#` prefix here 🤔 WDYT?

Reply:
[doc.1](https://doc.rust-lang.org/rust-by-example/compatibility/raw_identifiers.html)
[doc.2](https://doc.rust-lang.org/book/appendix-01-keywords.html#raw-identifiers)

According to the documentation, r# keywords can even be used as function names 😆

Reply:
> According to the documentation, r# keywords can even be used as function names

oh~ you're right. I didn't know this rust spec. please just ignore my comment~!